invenio_github/views/github.py

```diff
@@ -132,7 +132,7 @@ def get_repository(repo_name):
             repo = github.get_repository(repo_name)
             latest_release = github.repo_last_published_release(repo)
             default_branch = (
-                github.account.extra_data.get("repos", [])
+                github.account.extra_data.get("repos", {})
```

Comment:
Not related with the fix of this issue, but `github.account.extra_data` is an object, not a list (we even do `.get()` afterwards which would blow up)